lib/tasks/payment_breakdown.rake

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+namespace :payment_calculation do
+  desc "run payment calculator for a given lead provider"
+  task breakdown: :environment do
+    logger = Logger.new($stdout)
+    logger.formatter = proc do |_severity, _datetime, _progname, msg|
+      "#{msg}\n"
+    end
+
+    begin
+      lead_provider = LeadProvider.find(ARGV[1])
```

Comment:
Instead of using `ARGV`, could we do 
```
task breakdown:, [:lead_provider_name] => :environment do |_task, args|
  lead_provider_name = args.lead_provider_name
```
or something similar?

Reply:
I personally prefer the ARGV as otherwise you will need to run it using the following:

```
rake "payment_calculation:breakdown[Capita]"
```

instead of a more intuitive:

```
rake payment_calculation:breakdown Capita
```

Reply:
Alright.

In that case, could we add a note to readme, or a comment in this file about running this task? Seeing how (again) some tasks use another parameter format?

Reply:
Add here 17f91d9b